lib/players/computer.rb

```diff
@@ -0,0 +1,52 @@
+require 'pry'
+module Players
+  class Computer < Player
+
+  WIN_COMBINATIONS = [
+    [0, 1, 2],
+    [3, 4, 5],
+    [6, 7, 8],
+    [0, 3, 6],
+    [1, 4, 7],
+    [2, 5, 8],
+    [0, 4, 8],
+    [2, 4, 6]
+  ]
+
+    def move(board)
+    	square = ""
+    	winning_square = self.two_in_combo?(board)
+    	if winning_square
+    		square = winning_square
+    	elsif board.cells[4] == " "
+    		square = "5"
+    	else
+    		until board.valid_move?(square)
+      			square = (1..9).to_a.sample
+      		end
+      	end
+      	square.to_s
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression. For example:
```
def move(board)
  puts "Where would you like to move?"
  gets.chomp
end
```